README.md

```diff
@@ -50,11 +64,16 @@ When a user clicks the checkbox, this triggers the `handleCheckBoxClick` functio
 
 ### That's Great, but What Does it Have to Do with Redux?
 
-State in Redux is similar to state in React but at a larger scale. While state in React is focused on an individual component and its children, **Redux holds the state for our whole application**. Redux state refers to both the data that our application needs to use in our layout **PLUS** the logic around how that data is displayed. For example, let's say that we're using our checkbox React component in an app used at the Flatiron School that keeps track of student progress. We need to keep track the students in the class and figure out if any of them haven't gotten approval from an instructor for their app idea.
+State in Redux is similar to state in React but at a larger scale. While state in React is focused on an individual component and its children, **Redux holds the state for our whole application**. Redux state refers to both the data that our application needs to use in our layout **PLUS** the logic around how that data is displayed. For example, let's say that we're using our checkbox React component in an app used at the Flatiron School that keeps track of student progress. We need to keep track of the students in the class and figure out if any of them haven't gotten approval from an instructor, for their app idea.
```

Comment:
Given that this curriculum is supposed to be for online and in person, how do you feel about changing the last sentence to something like: "We need to keep track of the students that have graduated." or something more generic like that?